test_scripts/API/VehicleData/GetVehicleData/011_GetVD_allowed_and_disallowed_after_PTU.lua

```diff
@@ -0,0 +1,72 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
In case to allow one parameter to be tested by defining VD_PARAMS environment variable. E.g. export VD_PARAMS=accPedalPosition  
```
Test is FAILED
 HMI call VehicleInfo.GetVehicleData: Timeout: HMI call VehicleInfo.GetVehicleData: Timeout expired
 Response to 5: ValidIf: 
payload.resultCode: expected: SUCCESS, actual value: DISALLOWED
```
I propose to handle this issue

Reply:
Unfortunately this couldn't be handled since all these new scripts assume one parameters is disabled and others are enabled. Hence these scripts have no sense for a single parameter.